pkg/api/http.go

```diff
@@ -39,6 +39,7 @@ const (
 	urlParamShard           = "shard"
 	urlParamShardCount      = "shardCount"
 	urlParamSince           = "since"
+	urlParamExemplars       = "exemplars"
```

Comment:
Is the plan to be a query hint or http param?  Don't think we need both.  Hints are passed along to queriers/generators already.

Reply:
It's configured via a parameter in the query-frontend and also can be enabled via a query hint. The HTTP param is only used to pass the decision to queriers and generators. It's ignored in query-frontends. What do you think of that approach?